File: app/src/main/java/com/rorpage/purtyweather/managers/ServiceManager.java
package com.rorpage.purtyweather.managers;

import android.content.Context;
import android.content.Intent;

import com.rorpage.purtyweather.services.UpdateWeatherService;

public class ServiceManager {
    public static void StartUpdateWeatherService(Context context) {
        Intent service = new Intent(context, UpdateWeatherService.class);
        context.startForegroundService(service);
    }
}

File: app/src/main/java/com/rorpage/purtyweather/managers/ServiceManager.kt
package com.rorpage.purtyweather.managers

import android.content.Context
import android.content.Intent
import com.google.accompanist.pager.ExperimentalPagerApi
import com.rorpage.purtyweather.services.UpdateWeatherService

@ExperimentalPagerApi
object ServiceManager {
    @JvmStatic
    fun startUpdateWeatherService(context: Context) {
        val service = Intent(context, UpdateWeatherService::class.java)
        context.startForegroundService(service)
    }
}